src/alliteratedActors.test.js

```diff
@@ -2,61 +2,41 @@
 
 const alliteratedActors = require('./alliteratedActors');
 
-test('No names', () => {
+test('Should work with empty array', () => {
   expect(alliteratedActors([]))
     .toEqual(undefined);
 });
 
-test('Some actors', () => {
+test('Should return only first concurrence', () => {
   expect(alliteratedActors(
     ['Tom Hanks', 'Julia Roberts', 'James Earl Jones',
       'Sylvester Stallone', 'Leonardo DiCaprio', 'Amy Adams']))
     .toEqual('Sylvester Stallone');
 });
 
-test('Some actors and movie directors', () => {
-  expect(alliteratedActors(
-    ['Harrison Ford', 'Robert De Niro', 'Morgan Freeman', 'Matt Damon',
-      'Christofer Nolan', 'Natalie Portman', 'Martin Scorsese', 'Nick Nolte']))
-    .toEqual('Nick Nolte');
-});
-
-test('Five people', () => {
+test('Should return undefined when none of the names is alliterated ', () => {
   expect(alliteratedActors(
     ['Fpzwpgpq Vswaed Eagbsm', 'Qv Owj', 'Wzucu Qmwrw', 'Djnrai Uvrwyh',
       'Naclcthdv Bpooabnvt Gwpfsy']))
     .toEqual(undefined);
 });
 
-test('Ten people', () => {
-  expect(alliteratedActors(['Wn Tjuyol', 'Eyd Gjvxav', 'Wjuybt Hc Fiwyr',
-    'Vpikwvhgj Kt', 'Epuq Jji', 'Ohp Vt', 'Ufqmup Gdqhwwrv',
+test('Should work with three-word name', () => {
+  expect(alliteratedActors(['Wn Tjuyol', 'Eyd Gjvxav', 'Hjuybt Hc Fiwyr',
```

Comment:
you can shorten this test case significantly to 
```
expect(alliteratedActors(['Ohp Op Ot', 'Vnnchyxwx Vxot'])
     .toEqual('Ohp Op Ot');
```

Reply:
done